### PR TITLE
[epic] Nucleus 3.0 changes

### PR DESCRIPTION
**Pull request by MekalaNagarajan-Centrica**
- [x] Update 3.0 changes
- [x] Include `Property` column in Spec section

### CODE REVIEW

**Review by jholt1 — APPROVED** [review]

File: docs/components/ns-illustration.md [hunk 1]
```diff
@@ -37,9 +37,10 @@ The ns-illustration component can be used in a number of components to help crea
...
+| Attribute | Property | Type | Default | Options | Description |
+| :--- | :--- | :--- | :--- | :--- | :--- |
+| `type` | `type` | `string` | `standard` | `standard` | Type of illustration. |
+| `name` | `name` | `string` | `gas` | `appliance`, `appliance-cover`, `appliance-repair`, `battery`, `boiler`, `boiler-cover`, `boiler-repair`, `bulb`, `calendar`, `central-heating`, `chat`, `clock`, `compass`, `cosmo`, `document`, `dual-fuel`, `electric-charging-station`, `electric-vehicle`, `electrics`, `engineer`, `error`, `fix`, `fixed-sa`, `gas`, `green-leaf`, `help`, `home`, `home-insurance`, `laptop`, `low-carbon`, `multiple-people`, `oven`, `phone`, `pipe-and-drain`, `protect`, `radiator`, `savings`, `settings`, `sms`, `tap`, `track-engineer`, `van`, `whats-app`, `yellow-shield` See [Illustration component in Storybook](https://main--63ea3fa8c3721b415537bccc.chromatic.com/?path=/story/components-ns-illustration--all) | Picks the illustration that is to be shown. |
 | `size` | `number` | `''` | `''`, `1`, `2`, `3`, `4`, `5` | Changes the size of the illustration, with 1 being the smallest. |
```
Comment (andij) [inline docs/components/ns-illustration.md:44]: Add additional cell now there's 6 rows

```suggestion
| `size` | `number` | `''` | `''`, `1`, `2`, `3`, `4`, `5` | | Changes the size of the illustration, with 1 being the smallest. |
```

File: docs/components/ns-illustration.md [hunk 2]
```diff
@@ -37,9 +37,10 @@ The ns-illustration component can be used in a number of components to help crea
...
 | `inactive` | `string` | `false` | `true`, `false` | Will grey out the illustration. |
```
Comment (andij) [inline docs/components/ns-illustration.md:45]: Add additional cell now there's 6 rows

```suggestion
| `inactive` | `string` | `false` | `true`, `false` | | Will grey out the illustration. |
```

File: docs/components/ns-inputter.md [hunk 3]
```diff
@@ -66,13 +66,13 @@ To match this heading's size with other `ns-inputter` labels', group all associa
...
+Checkboxes are usually the answers to a question asked in the `heading` attribute of the `ns-inputter` - such as "Which British Gas services would you like to choose?".
 
 #### Note on the size of the heading for checkboxes
 
 Checkboxes come with a heading that inherits size 4, which is larger than the default label size and can visually look odd when grouped with other `ns-inputters`.
 
-To match this heading's size with other `ns-inputter` labels', group all associated elements in an `ns-fieldset`. This will help keep a logical visual hierarchy when using checkboxes grouped with other inputs.
+To match this heading's size with other `ns-inputter` labels', group all associated elements in an `ns-form-group`. This will help keep a logical visual hierarchy when using checkboxes grouped with other inputs.
 
 ![Checkboxes](images/ns-inputter/checkboxes.webp)
```
Comment (andij) [inline docs/components/ns-inputter.md:77]: This checkbox image should ideally have one checkbox checked.

There looks like there is a suitable screenshot here: https://epic-nucleus-3-0.nucleus-docs.pages.dev/docs/patterns/asking-for-information#radio-buttons-and-checkboxes

File: docs/components/ns-password.md [hunk 4]
```diff
@@ -93,14 +88,26 @@ Also see ns-inputter [specification](components/ns-inputter.md#specification)
 | Attribute | Type | Default | Options | Description |
 | :--- | :--- | :--- | :--- | :--- |
 | `type` |  `string`  | `default` | [`default`, `confirm`] | The type of the password component. Use `default` type for login journey and `confirm` type for registration journey. |
+| `name` | `string` | `password` |  | See [ns-inputter](components/ns-inputter.md#specification). |
 | `value`   | `string` | `''` |  | The value of the input or group of inputs inside `ns-inputter`. This can be used to add an initial value. |
 | `helper` | `string` |  |  | See [ns-inputter](components/ns-inputter.md#specification). |
-| `tip` | `string` |  |  | See [ns-inputter](components/ns-inputter.md#specification). |
-| `name` | `string` | `password` |  | See [ns-inputter](components/ns-inputter.md#specification). |
-| `passwordValidation` | `array` | `['hasNumbers', 'hasLetters', 'isBetween(8, 20)']` | |  Assists a user when adding a valid password and shows an error message when they're not. See [ns-inputter](components/ns-inputter.md#validation)
```
Comment (andij) [inline docs/components/ns-password.md:100]: `passwordValidation` has been removed, is this correct?

Comment (MekalaNagarajan-Centrica) [reply docs/components/ns-password.md:100]: Yes. That's correct.

File: docs/components/ns-inputter.md (hunk 3 above)
Comment (andij) [reply docs/components/ns-inputter.md:77]: Sorry, my mistake. I observed a difference between this PR and live, and noted it as an issue.

File: docs/components/ns-illustration.md (hunk 2 above)
Comment (MekalaNagarajan-Centrica) [reply docs/components/ns-illustration.md:45]: Fixed in PR https://github.com/centrica-engineering/nucleus-docs/pull/242/files

File: docs/components/ns-illustration.md (hunk 1 above)
Comment (MekalaNagarajan-Centrica) [reply docs/components/ns-illustration.md:44]: Fixed in PR https://github.com/centrica-engineering/nucleus-docs/pull/242/files

**Review by andij — APPROVED** [review]

**Pull request merged**